docs/src/literate-tutorials/reactive_surface.jl

```diff
@@ -0,0 +1,259 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+if isdefined(Main, :is_ci) #hide
+    IS_CI = Main.is_ci     #hide
+else                       #hide
+    IS_CI = false          #hide
+end                        #hide
```

Comment:
```suggestion
end                        #hide
nothing                    #hide
```
To hide the `false` printout in https://ferrite-fem.github.io/Ferrite.jl/previews/PR715/tutorials/reactive_surface/